integration/deploy/k8s_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build integration
+// +build integration
+
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/okteto/okteto/integration"
+	"github.com/okteto/okteto/integration/commands"
+	"github.com/stretchr/testify/require"
+)
+
+// TestDeployPipelineFromK8s tests the following scenario:
+// - Deploying a pipeline manifest locally from a k8s file
+// - The endpoints generated are accessible
+func TestDeployPipelineFromK8s(t *testing.T) {
```

Comment:
Why pipeline if it is using `okteto deploy` command?

Reply:
The term we use now is dev environment. Using pipeline might be confusing because the reader could think that we are deploying the app using `okteto pipeline deploy` with a repo but we are using `okteto deploy` if I'm not wrong. 

I want to avoid the confusion `okteto pipeline deploy` vs `okteto deploy`